tests/fixtures/sample_lib/modules/file_processing/bounding_box_module.py

```diff
@@ -0,0 +1,31 @@
+"""
+A sample module for sample things!
+"""
+# Standard
+
+# Local
+from ...data_model.sample import FileTask, FileDataType
+from caikit.core.modules import ModuleLoader
+import caikit.core
+
+
+@caikit.core.module(
+    "750cad4d-c3b8-4327-b52e-e772f0d6f311", "BoundingBoxModule", "0.0.1", FileTask
+)
+class BoundingBoxModule(caikit.core.ModuleBase):
```

Comment:
This is my first time creating a caikit module so please let me know if I missed anything or didn't follow best practices

Reply:
This looks great! A very nice simple dummy module implementation